analytics/config.py

```diff
@@ -21,8 +21,8 @@
     merge_enabled= True,
     # add validators for our required config vars
     validators=[
-        Validator("SLACK_BOT_TOKEN", must_exist=True),
-        Validator("REPORTING_CHANNEL_ID", must_exist=True),
+        Validator("SLACK_BOT_TOKEN", must_exist=False), #disabled for testing
+        Validator("REPORTING_CHANNEL_ID", must_exist=False), #disabled for testing
```

Comment:
Billy wrote some end to end tests that require these tokens. So these need to stay enabled if we want to run the full end to end test suite, unfortunately.

That said, I don't think we should have ever written end to end tests that require these tokens to be set. So I'm willing to accept this change.